bin/generate.py

```diff
@@ -1035,6 +1057,7 @@ def generate(d, problem, generator_config, bar):
             bar.log(f'REMOVED: testdata.yaml')
         bar.done()
 
+    def generate_includes(d, problem, generator_config, bar):
```

Comment:
Re. the other comment: here you split `generate` into `generate_includes`. Shouldn't the `copy_generated` bit live inside here, as further down you write:
```
        def generate_copies_and_includes(d):
            p.join()
            d.generate_includes(self.problem, self, bar)
```

Reply:
copy generated is the function that moves files from tmp to data (the function was always there)